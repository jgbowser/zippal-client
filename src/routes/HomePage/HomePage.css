.welcome {
  padding: 0 10px;
}

.welcome p {
  font-size: 18px;
}
