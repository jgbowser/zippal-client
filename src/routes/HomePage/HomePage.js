import React, {Component} from 'react'

export default class HomePage extends Component {
  render(){
    return(
        <div className='welcome'>
          <h2>Welcome to Zip Pal!</h2>
          <br/>
          <h3>what is Zip Pal?</h3>
          <p>Zip Pal was created out of that familure human need for connections. In these troubled times, it is so hard to just meet people.<br/>
          That is where Zip Pal comes in! With Zip Pal, you can start a conversation with another random user anytime you'd like! </p>
        </div>
    )
  }
}