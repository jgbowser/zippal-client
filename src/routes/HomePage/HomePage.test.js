import React from 'react'
import ReactDOM from 'react-dom'
import HomePage from './HomePage'
import renderer from 'react-test-renderer'

describe('HomePage Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<HomePage />, div)
    ReactDOM.unmountComponentAtNode(div)
  })
  it(`takes snapshot of HomePage component`,()=>{
    const tree = renderer
    .create(<HomePage/>)
    expect(tree).toMatchSnapshot()
  })
})