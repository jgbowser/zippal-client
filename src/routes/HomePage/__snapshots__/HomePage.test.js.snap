// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Component takes snapshot of HomePage component 1`] = `
<div
  className="welcome"
>
  <h2>
    Welcome to Zip Pal!
  </h2>
  <br />
  <h3>
    what is Zip Pal?
  </h3>
  <p>
    Zip Pal was created out of that familure human need for connections. In these troubled times, it is so hard to just meet people.
    <br />
    That is where Zip Pal comes in! With Zip Pal, you can start a conversation with another random user anytime you'd like! 
  </p>
</div>
`;
